homeassistant/const.py

```diff
@@ -188,7 +188,11 @@
 STATE_NOT_HOME = 'not_home'
 STATE_UNKNOWN = 'unknown'
 STATE_OPEN = 'open'
+STATE_OPENED = 'opened'
```

Comment:
Just use `STATE_OPEN` here. We don't need to define an extra constant for this. It's OK that it maps to `"Opened"` on the Z-Wave side. That's why we have the mapping.